mff_rams_plugin/config.py

```diff
@@ -8,3 +8,15 @@ class ExtraConfig:
     @property
     def DEALER_BADGE_PRICE(self):
         return self.get_attendee_price()
+
+    @property
+    def TABLE_OPTS(self):
+      return [ (1.0, 'Single Table'),(2.0, 'Double Table'),(3.0, 'Triple Table'),(4.0, 'Quad Table'),(5.0, '10x10 Booth')]
+
+    @property
+    def ADMIN_TABLE_OPTS(self):
+      return [(0.0, 'No Table')] + TABLE_OPTS()
```

Comment:
This should be `c.TABLE_OPTS` since it's a property we're accessing from the config class.